src/components/inputs/button/button.stories.tsx

```diff
@@ -201,3 +207,35 @@ export const IconButtons: ComponentStory<typeof Button> = () => {
     </div>
   );
 };
+
+/**
+ * File upload buttons used with or without icons.
+ */
+export const UploadButton: ComponentStory<typeof Button> = () => {
+  return (
+    <div className="story__button">
+      <input
+        accept="image/*"
+        id="icon-button-file"
+        multiple
```

Comment:
`Multiple` should become a prop. You don't always want to let the user select multiple files.

Reply:
Done